content/ui-patterns/degraded-experiences.mdx

```diff
@@ -36,7 +36,7 @@ GitHub's internal article about graceful degradation describes the difference be
 
 <div>
 
-If there is a critical system error that will degrade the user experience, show a [flash](/components/flash) banner at the top of the page above the global navigation. Having a global banner helps set the expectation that some parts of the usual UI might be missing or broken. Default to using the `"warning"` variant of the flash.
+If there is a critical system error that will degrade the user experience, show a [banner](/components/banner) at the top of the page above the global navigation. Having a global banner helps set the expectation that some parts of the usual UI might be missing or broken. Default to using the `"warning"` variant of the banner.
```

Comment:
For this part:

> at the top of the page above the global navigation

Does this mean we need a full width option or is it okay to have its current style?

Reply:
I think this is meant to be the `Announcement` component but I'm pretty confused about the state of that proposal 😅 @lukasoppermann @dipree